godal.cpp

```diff
@@ -470,7 +470,28 @@ GDALDatasetH godalDatasetWarp(char *dstName, GDALDatasetH ds, char **switches, c
 		return nullptr;
 	}
 	int usageErr=0;
-	GDALDatasetH ret = GDALWarp(dstName, nullptr, 1, &ds, warpopts, &usageErr);
+	GDALDatasetH ret = GDALWarp(dstName, nullptr, nSrcCount, srcDS, warpopts, &usageErr);
+	GDALWarpAppOptionsFree(warpopts);
+	godalUnwrap(config);
+	if(*error!=nullptr) {
+		return nullptr;
+	}
+	if(ret==nullptr || usageErr!=0) {
+		*error=strdup("warp: unknown error");
+	}
+	return ret;
+}
+
+GDALDatasetH godalDatasetWarpInto(GDALDatasetH dstDs,  int nSrcCount, GDALDatasetH *srcDS, char **switches, char **error, char **config) {
```

Comment:
To remain consistent with other functions here, this should be
```c++
char* godalDatasetWarpInto(GDALDatasetH dstDs,  int nSrcCount, GDALDatasetH *srcDS, char **switches, char **config)
```
i.e. the error message is the return value